### DISCUSSION TITLE
Why was the `materialized_postgresql_allow_automatic_update` setting for MaterializedPostgreSQL database removed?

### DISCUSSION THREAD

**Discussion by B3zaleel** [ClickHouse/ClickHouse#56439] [General]
This setting seems to provide simplicity in synchronizing two databases. Detaching and attaching the tables to the database can be quite tedious. Is there an alternative way to go about synchronizing Postgres with ClickHouse?

**Comment by alexey-milovidov** [comment]
ClickHouse has `MaterializedPostgreSQL` under preview, which means:
- it is still in development, and we can change its behavior or remove it completely;
- you acknowledged it by unlocking an experimental feature with a setting `allow_experimental_database_materialized_postgresql`;

**Comment by tbragin** [comment]
> Is there an alternative way to go about synchronizing Postgres with ClickHouse?

You could try a 3rd party CDC tool like Debezium https://clickhouse.com/blog/clickhouse-postgresql-change-data-capture-cdc-part-1

**Comment by coffenbacher** [comment]
Hi all, also curious about this setting for the same reason: "Detaching and attaching the tables to the database can be quite tedious.". 

Understood that there are alternatives (I've written my own CDC tool in the past) but I am curious if there's going to be a new way to handle simple schema updates going forward.
